packages/jsii-pacmak/test/generated-code/examples/nested-types.ts

```diff
@@ -0,0 +1,31 @@
+// See: https://github.com/aws/jsii/issues/2649
```

Comment:
wondering if perhaps we should afford for multiple files so basically each example will be in a separate directory?

Reply:
We could update it so that - if it's a file, it's a single-file example... If it's a directory, it's a multiple-files example. I went for just what I needed at this point.

Reply:
Okay, we can iterate on this